api/app/routes/admin/feedback.py

```
@@ -41,9 +38,18 @@
     },
 )
 
-# Create singleton instances
-feedback_service = FeedbackService(settings=settings)
-faq_service = FAQService(settings=settings)
+# Initialize settings and services with fallback for testing
+try:
+    settings = get_settings()
+    feedback_service = FeedbackService(settings=settings)
+    faq_service = FAQService(settings=settings)
+except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)